erpnext_com/www/docs/user/manual/en/accounts/invoice_discounting.md

```diff
@@ -0,0 +1,52 @@
+<!-- add-breadcrumbs -->
+
+**Invoice discounting is the practice of using a company's unpaid invoices as collateral for a short term loan, which is issued by a bank or a finance company.**
+
+### 1. Prerequisite
+
+You need to create below ledgers in order to post invoice discounting transactions.
+
+**Short Term Loan:** A ledger under current liabilities for loan.
```

Comment:
```suggestion
**Short Term Loan:** A ledger under the 'Current Liabilities' account for loan.
```

Reply:
I am not referring to the 'Current Liabilities' group specifically...

Reply:
what are you referring to? the sentence is not clear for the end user